test/integration/user_cant_add_inactive_items_to_cart_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.